### ISSUE TITLE
Module information in the wiki? [Request]

### ISSUE THREAD

**Issue by Pecant-Pie** [gegy1000/PracticalSpaceFireworks#16]
I would greatly appreciate even a brief overview of what the different modules are capable of and any use cases, because this mod seems very fun and I would like to use it in my modpack.  Are you planning on adding that to the wiki page?

**Comment by tterrag1098** [comment]
Of course, I would love to document more things. But time is short right now.
